### ISSUE TITLE
The process was terminated due to an unhandled exception.

### ISSUE THREAD

**Issue by 0xNinshu** [microsoft/vscode-powerquery-sdk#291]
### Preflight Checklist

- [X] I have installed the [latest version of Power Query SDK](https://github.com/microsoft/vscode-powerquery-sdk/releases/latest).
- [X] I have checked existing resources, including the [common issues](https://learn.microsoft.com/en-us/power-query/commonissues) and the [release notes](https://github.com/microsoft/vscode-powerquery-sdk/releases).
- [X] I have searched for [similar issues](https://github.com/microsoft/vscode-powerquery-sdk/issues).

### Power Query SDK

0.2.3

### Platform

Windows

### Architecture

x64

### OS Version

Windows 11 Pro - Version: 22H2, Build: 22621.2506

### VSCode version

1.84.1

### PQSdkTool Path

c:\Users\<user>\.vscode\extensions\powerquery.vscode-powerquery-sdk-0.2.3-win32-x64\.nuget\Microsoft.PowerQuery.SdkTools.2.114.4\tools

### Bug Description

I noticed that this issue was in my event log. I was not using the vscode extension, and for some reason the PQServiceHost.exe is running in the background and failing. This event appears multiple times within a day.
```
<Event xmlns="http://schemas.microsoft.com/win/2004/08/events/event">
<System>
  <Provider Name=".NET Runtime" /> 
  <EventID Qualifiers="0">1026</EventID> 
  <Version>0</Version> 
  <Level>2</Level> 
  <Task>0</Task> 
  <Opcode>0</Opcode> 
  <Keywords>0x80000000000000</Keywords> 
  <TimeCreated SystemTime="2023-11-05T18:26:48.9112643Z" /> 
  <EventRecordID>95982</EventRecordID> 
  <Correlation /> 
  <Execution ProcessID="30460" ThreadID="0" /> 
  <Channel>Application</Channel> 
  <Computer>ASUS-DESKTOP</Computer> 
  <Security /> 
  </System>
<EventData>
  <Data>Application: PQServiceHost.exe Framework Version: v4.0.30319 Description: The process was terminated due to an unhandled exception. Exception Info: System.ComponentModel.Win32Exception at System.Diagnostics.ProcessManager.OpenProcess(Int32, Int32, Boolean) at System.Diagnostics.Process.GetProcessHandle(Int32, Boolean) at System.Diagnostics.Process.get_HasExited() at PQServiceHost.Program.RunningInstance() at PQServiceHost.Program.Main(System.String[])</Data> 
  </EventData>
  </Event>
```

### Steps to Reproduce

I am unsure of how to repro the issue, since I was not using the power query sdk / vscode extension when it happened.

### Actual Experience

I am unsure of how to repro the issue, since I was not using the power query sdk / vscode extension when it happened.

**Comment by mattmasson** [comment]
Thank you for the report - I believe I've found the problem.

In the meantime, you can turn off the use of the PQTest service host in the configuration options:

configuration option id: `powerquery.sdk.features.useServiceHost`

![image](https://github.com/microsoft/vscode-powerquery-sdk/assets/5509937/2a3ca4c2-efa7-4d94-a029-3d6370d1f81a)

**Comment by mattmasson** [comment]
This should be resolved in the [0.3.1 release](https://github.com/microsoft/vscode-powerquery-sdk/releases/tag/v0.3.1) (sdktools `2.127.2`).
